tests/libdv/dv.go

```diff
@@ -133,3 +148,60 @@ func WithForceBindAnnotation() dvOption {
 func randName() string {
 	return "test-datavolume-" + rand.String(dvRandomNameLength)
 }
+
+// PVC Options
+
+// PVCWithStorageClass add the sc storage class name to the DV
+func PVCWithStorageClass(sc string) pvcOption {
```

Comment:
Let's leave the PVC* from these options. We should have type safety.

Reply:
The compiler will protect us here, but it will be not intuative and misleading forvthe user of this package, to have two sets of With* options. This is why I  added the PVC prefix.

Reply:
I think the same way as @nunnatsa

Reply:
Ok, I take it